Data_Structures_Answers.md

```diff
@@ -2,17 +2,33 @@ Add your answers to the questions below.
 
```

Comment:
Answers all check out Michael 👍 , good job explaining your answers in a concise way.